notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        ! gsutil rm -r $BUCKET_NAME
</pre><p>Please add the extra if statement in the notebook template that checks if it's in test mode, before deleting the bucket.</p><p> </p><p>We want to be careful that the user doesn't accidentally delete their entire bucket, if they have other files in it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>I have added the check of 'IS_TESTING' before deleting the bucket. </p>